Tests/ProfilerExtension/ProfilerExtensionTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace EmanueleMinotto\TwigCacheBundle\Tests\DependencyInjection;
+
+use EmanueleMinotto\TwigCacheBundle\Tests\AppKernel;
+use EmanueleMinotto\TwigCacheBundle\Twig\ProfilerExtension;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @coversDefaultClass \EmanueleMinotto\TwigCacheBundle\Twig\ProfilerExtension
+ */
+class ProfilerExtensionTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \Symfony\Component\HttpKernel\Kernel
+     */
+    private $kernel;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        $this->kernel = new AppKernel('TwigCacheExtensionTest', true);
+        $this->kernel->boot();
+    }
+
+    /**
+     * @covers ::load
```

Comment:
Travis is giving a problem because this test doesn't cover a `\EmanueleMinotto\TwigCacheBundle\Twig\ProfilerExtension::load` method.

Replace it with:
```
    /**
     * @covers ::serialize
     * @covers ::unserialize
     */
```